test/functional/mempool_accept.py

```diff
@@ -4,6 +4,8 @@
 # file COPYING or http://www.opensource.org/licenses/mit-license.php.
 """Test mempool acceptance of raw transactions."""
 
+
+from decimal import Decimal
```

Comment:
19940: this is like super-partial :D  the fix should be simply a part of 07c6897cae1875c7ad88bc3127bf26541b74c8ff (20039 backport)

Reply:
:) Sure will make it a part of 20039 ,